_data/past.yml

```diff
@@ -140,6 +169,7 @@
   dates: "June 3-5, 2020"
   url: https://nordictestingdays.eu/?utm_source=testingconferences
   twitter: nordictestdays
+  status: Postponed
```

Comment:
We remove statuses for past conferences

Reply:
That's true, I removed all but Cancelled & Postponed... because they kinda didn't happen... So a bit misleading. But can remove that too

Reply:
fixed